libobs/obs-source.c

```diff
@@ -2391,12 +2389,10 @@ static void obs_source_update_async_video(obs_source_t *source)
 				source->timing_set = true;
 			}
 
-			if (source->async_update_texture) {
+			if (async_update_texture)
```

Comment:
Add the braces back. Even though it's a single statement, it's still multiple lines. The braces make it easier to understand.

Reply:
I got it.